libs/@hashintel/petrinaut/dev/vite.config.ts

```diff
@@ -0,0 +1,13 @@
+import react from "@vitejs/plugin-react";
+import { defineConfig } from "vite";
+
+export default defineConfig({
+  plugins: [react()],
+  root: "dev",
+  server: {
+    open: true,
+  },
+  define: {
+    "process.env.NODE_ENV": '"development"',
```

Comment:
You normally do not need that with Vite.

Reply:
Thanks, removed!